gamemode/core/hooks/sv_hooks.lua

```diff
@@ -694,6 +694,9 @@ function GM:NutScriptTablesLoaded()
 		:catch(ignore)
 	nut.db.query("ALTER TABLE nut_players ADD COLUMN _lastJoin DATETIME")
 		:catch(ignore)
+	-- Add missing _quantity column for nut_item table.
+	nut.db.query("ALTER TABLE nut_items ADD COLUMN _quantity INTEGER")
```

Comment:
Have you tested this with MySQL? I don't think `INTEGER` is a MySQL type.

Reply:
Yes I have, INT or INTEGER works. https://dev.mysql.com/doc/refman/8.0/en/integer-types.html